apis/concierge/authentication/v1alpha1/types_jwt.go.tmpl

```diff
@@ -27,6 +27,16 @@ type JWTAuthenticatorSpec struct {
 	// +kubebuilder:validation:MinLength=1
 	Audience string `json:"audience"`
 
+    // UsernameClaim is the name of the claim which should be read to extract the
+    // username from the JWT token. When not specified, it will default to "username".
+	// +optional
+    UsernameClaim string `json:"username_claim"`
```

Comment:
I just took a peak at this because I was triggering CI for this PR...should we consider following the same `Spec.Claims.Username`/`Spec.Claims.Groups` pattern that we follow in the `UpstreamOIDCProvider` so that our API is consistent with respect to claim configuration?

https://github.com/vmware-tanzu/pinniped/tree/main/generated/1.19#k8s-api-go-pinniped-dev-generated-1-19-apis-supervisor-idp-v1alpha1-oidcclaims

Reply:
Resolved in the most recent commit. Thanks!